packages/components/header/template.njk

```diff
@@ -136,7 +136,7 @@
     
     <div class="nhsuk-navigation-container">
       <nav class="nhsuk-navigation" id="header-navigation" role="navigation" aria-label="Primary navigation">
-        <ul class="nhsuk-header__navigation-list">
+        <ul class="{{ "nhsuk-header__navigation-list nhsuk-header__navigation-list--left-aligned" if params.primaryLinks.length < 4 else "nhsuk-header__navigation-list" }}">
```

Comment:
I wonder if this could be refactored so `nhsuk-header__navigation-list` was always included, and it's just the second class that's conditional?